app/components/Navbar/Navbar.tsx

```diff
@@ -2,11 +2,14 @@ import { Link } from "@remix-run/react";
 import GitHubIcon from "../icons/GitHubIcon";
 import MeetupIcon from "../icons/MeetupIcon";
 import YouTubeIcon from "../icons/YouTubeIcon";
+import DiscordIcon from "../icons/DiscordIcon";
 
 export const homepageLinkTitle = "Remix Austin 💿";
 export const meetupUrl = "https://www.meetup.com/remix-austin/";
 export const gitHubUrl = "https://github.com/remix-austin";
 export const youTubeUrl = "https://www.youtube.com/@remixaustin";
+export const discordUrl =
```

Comment:
Should this be the invite link?  If we want it to come from Brooks with his name, does he need to create a unique invite url?  Should we keep using this channel url, but explain how to get invited to the Remix server in the card below or something?

cc: @brookslybrand

Reply:
> If we want it to come from Brooks with his name, does he need to create a unique invite url?
That's my understanding, yes.

> Should we keep using this channel url, but explain how to get invited to the Remix server in the card below or something?
I'm not super knowledgeable on all things Discord. I wanna say the invite link is the right way to go and Discord will do the explaining 🤷🏻

Reply:
So sorry, y'all have been moving way faster than I've been able to (which is awesome!)

Try this link? https://discord.gg/mYy2kzqX